core/src/script/CGXP/plugins/Profile.js

```diff
@@ -533,15 +535,14 @@ cgxp.plugins.Profile = Ext.extend(gxp.plugins.Tool, {
     },
 
     showMarker: function(row) {
-        var layers = this.rasterLayers;
         var datum = this.data[row];
         var point = new OpenLayers.Geometry.Point(datum.x, datum.y);
         this.marker && this.marker.destroy();
 
         var label = [datum.dist + " m"];
-        for (var i=0; i < layers.length; i++) {
-            var l = layers[i];
-            label.push(l + " : " + datum[this.valuesProperty][l] + " m");
+        for (var i = 0 ; i < this.rasterLayers.length ; i++) {
+            label.push(this.i18nRasterLayers[i] + " : " +
```

Comment:
Good question,
Actually in french we add it and it's the most used language ...
I can change to make it internationalisable :-)